bundles/org.openhab.binding.doorbird/NOTICE

```diff
@@ -0,0 +1,25 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab2-addons
+
+== Third-party Content
+
+lazysodium-java
```

Comment:
If we do not include but only use the libraries you can remove that.

Reply:
Ah, ok. I didn't understand the distinction. Removed.